netcode.c

```diff
@@ -41,10 +41,10 @@
 #define NETCODE_SOCKET_IPV6         1
 #define NETCODE_SOCKET_IPV4         2
 
-#define NETCODE_CONNECT_TOKEN_PRIVATE_BYTES 1024
-#define NETCODE_CHALLENGE_TOKEN_BYTES 300
+#define NETCODE_CONNECT_TOKEN_PRIVATE_BYTES ( 1024 + 8 )
+#define NETCODE_CHALLENGE_TOKEN_BYTES ( 300 + 8 )
 #define NETCODE_VERSION_INFO_BYTES 13
-#define NETCODE_USER_DATA_BYTES 256
+#define NETCODE_USER_DATA_BYTES ( 256 + 8 )
```

Comment:
What are these +8 for? Is that needed? I also see some other +8 in netcode.c as well.

Reply:
I have that to distinguish what netcode.io uses for their default size, then what we are adding.  In this case its only 8 bytes so far.  But in some other spots there is different reasoning...

Reply:
Actually this one isn't needed.  Removing it.